src/Pages/UserListTable.js

```diff
@@ -0,0 +1,148 @@
+import { Button, Table, TableBody, TableContainer } from "@mui/material";
+import EditRoundedIcon from "@mui/icons-material/EditRounded";
+import DeleteOutlineTwoToneIcon from "@mui/icons-material/DeleteOutlineTwoTone";
+import { TableHead, TableRow, Paper } from "@mui/material";
+import TableCell, { tableCellClasses } from "@mui/material/TableCell";
+import { deleteRecord, fetchAirline, fetchRecord } from "../Store/action";
+import { useHistory } from "react-router-dom";
+
+const StyledTableCell = styled(TableCell)(({ theme }) => ({
+  [`&.${tableCellClasses.head}`]: {
+    backgroundColor: theme.palette.common.black,
+    color: theme.palette.common.white,
+  },
+  [`&.${tableCellClasses.body}`]: {
+    fontSize: 14,
+  },
+}));
+
+const StyledTableRow = styled(TableRow)(({ theme }) => ({
+  "&:nth-of-type(odd)": {
+    backgroundColor: theme.palette.action.hover,
+  },
+
+  "&:last-child td, &:last-child th": {
+    border: 0,
+  },
+}));
+
+const UserListTable = () => {
+  const { getrecords } = useSelector((state) => state);
+
+  const [page, setPage] = React.useState(1);
+
+  let history = useHistory();
+
+  let dispatch = useDispatch();
+
+  useEffect(() => {
+    dispatch(fetchRecord());
+  }, [dispatch]);
+
+  useEffect(() => {
+    dispatch(fetchAirline());
+  }, [dispatch]);
+
+  useEffect(() => {
+    let params = {
+      page: "1",
+      size: "10",
+    };
+    let queryparams = queryString.stringify(params);
+    dispatch(fetchRecord(queryparams));
+  }, [dispatch]);
+
+  const handleDeleteRecord = (id) => {
+    window.confirm(`are you sure you want to delete ${id} ?`);
+    dispatch(deleteRecord(id));
+  };
+
+  const pageChangeHandler = (event, value) => {
+    let params = {
+      page: value,
+      size: "10",
+    };
+    let queryparams = queryString.stringify(params);
+    dispatch(fetchRecord(queryparams));
+    setPage(value);
+  };
+
+  return (
+    <Paper elevation={4} sx={{ mt: 1, mb: 3 }}>
+      <TableContainer component={Paper}>
+        <Button
+          color="primary"
+          aria-label="add"
+          variant="contained"
+          onClick={() => history.push("/addRecord")}
+          sx={{
+            mr: 5,
+            mt: 2,
+            mb: 2,
+            float: "right",
+          }}
+        >
+          Add New Record
+        </Button>
+        <Table sx={{ minWidth: 700 }} aria-label="customized table">
+          <TableHead>
+            <TableRow>
+              <StyledTableCell>Name</StyledTableCell>
+              <StyledTableCell align="center">Trips</StyledTableCell>
+              <StyledTableCell align="center">Id</StyledTableCell>
+              <StyledTableCell align="center">Airline Id</StyledTableCell>
+              <StyledTableCell align="center">Airline Name</StyledTableCell>
+              <StyledTableCell align="center">Action</StyledTableCell>
+            </TableRow>
+          </TableHead>
+          <TableBody>
+            {getrecords &&
```

Comment:
getrecords are array if you want to check it with conditionally then etrecords.length like this you can do it